src/Components/Tooltip/Tooltip.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import ReactTooltip from "react-tooltip";
+
+import "./Tooltip.scss";
+
+const Tooltip = ({
```

Comment:
could we add an id to connect the tooltip to the specific wrapper? [see](https://github.com/wwayne/react-tooltip#3-tooltip-not-binding-to-dynamic-content)